dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/ApiTokenResource.java

```diff
@@ -290,7 +291,10 @@ public final Response getRemoteToken(@Context final HttpServletRequest httpReque
                 }
             }
 
-            return Response.ok(response.readEntity(String.class)).build();
+            return Response
```

Comment:
I think a test can be included

Reply:
it already have failing test

https://raw.githack.com/dotCMS/test-results/master/projects/core/current/curl/postgres/reports/html/ApiToken_Resource.html